models.js

```diff
@@ -10,3 +10,14 @@ exports.selectCategories = () => {
       return res.rows;
     });
 };
+
+exports.findReviewByID = (id) => {
+  const arr = [parseInt(id.review_id)];
+  return connection
+    .query(`SELECT * FROM reviews WHERE review_id = $1`, arr)
+    .then((res) => {
+      return res.rows.length === 0
+        ? Promise.reject({ status: 404, msg: "Invalid input!" })
```

Comment:
As this is a custom error (which is totally fine and what I would do as well) consider the message to the user, is it an invalid input for a 404??